src/checkWhichHost/index.js

```diff
@@ -0,0 +1,18 @@
+export function checkWhichHost (URL) {
+  if (URL.startsWith('https://www.curseforge.com/wow/addons/')) {
+    return initCurseforgeObj(URL)
+  }
+
+  const errorObj = { 'error': "ERROR: Invalid URL '" + URL + "'. Given link does not match parse" }
+  return errorObj
+}
+
+function initCurseforgeObj (URL) {
```

Comment:
Same function

Reply:
what do you mean?

Reply:
@zainkai Referring to how this function and the other one I marked are pretty much exactly the same function, but only one is called, I'll leave it up to you how you want to organize how they're called/what to keep

Reply:
If that's the case we should be using parseUrl. We might want to look into creating an addon class to keep track of properties. But for the time being replace initCurseforgeObj with parseUrl and we can do those changes later down the line.